src/sidebar/components/group-list-item-out-of-scope.js

```diff
@@ -1,37 +1,106 @@
 'use strict';
 
+const classnames = require('classnames');
+const { Fragment, createElement } = require('preact');
+const { useState } = require('preact/hooks');
```

Comment:
`useState` is one way of persisting local state between renders. Here it is used to track the expanded/collapsed state. [See docs](https://reactjs.org/docs/hooks-state.html).

Reply:
Yes. Something I'm not sure about at the moment is whether there will be use cases for which we'll still need to use classes or are better done with classes. This is something we'll need to figure out as we go.